src/display/api.js

```diff
@@ -1409,7 +1411,6 @@ class PDFPageProxy {
     background = null,
     optionalContentConfigPromise = null,
     annotationCanvasMap = null,
-    pageColors = null,
```

Comment:
Why are we removing this functionality from the API itself, since it seems that it'll outright break things for third-party users?
For one thing, I'd be somewhat surprised if this doesn't also break the reference-tests that set custom `pageColors`.

Can't we keep this parameter, and apply any custom `pageColors` (using the new filter approach) as the *last step* in e.g. the `CanvasGraphics.endDrawing` method instead?